cves/CVE-2019-9721.yml

```diff
@@ -266,8 +277,8 @@ i18n:
 
     Answer should be boolean. Write a note about how you came to the conclusions
     you did.
-  answer:
-  note:
+  answer: false
+  note: This vulnerability is a result of a particularly crafted file in the Matroska format
```

Comment:
And the effected file was related to HTML subtitle parsing, which is highly related to i18n. Did the input to the malicious file contain any i18n text?